tests/ts/bazel_repository_test_dir/WORKSPACE

```diff
@@ -0,0 +1,71 @@
+workspace(name = "bazel_repository_test")
```

Comment:
Could you add a brief README to `tests/ts/bazel_repository_test_dir`? May be as simple as just pointing people to `tests/ts/bazel_repository_test.sh`, but I could see someone not familiar with bazel being confused.

Reply:
Done